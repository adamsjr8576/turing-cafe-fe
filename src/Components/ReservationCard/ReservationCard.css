.reservation-card {
  width: 250px;
  height: 225px;
  background-color: white;
  border: 5px solid skyblue;
  margin: 10px 0;
  border-radius: 10px;
}

.delete-button {
  width: 75%;
  padding: 5px;
  font-size: 1em;
  background-color: lightblue;
  border-radius: 10px;
  cursor: pointer;
}

.delete-button:hover {
  transform: scale(1.1);
}
