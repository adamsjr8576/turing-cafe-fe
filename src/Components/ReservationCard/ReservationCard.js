import React from 'react';
import './ReservationCard.css';

const ReservationCard = ({ date, id, name, number, time }) => {
  return (
    <article className='reservation-card'>
      <h2>{name}</h2>
      <p>{date}</p>
      <p>{time} PM</p>
      <p>Number Of Guest: {number}</p>
      <button className='delete-button'>Cancel</button>
    </article>
  )
}

export default ReservationCard;
