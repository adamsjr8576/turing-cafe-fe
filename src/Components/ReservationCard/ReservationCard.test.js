import React from 'react';
import ReactDOM from 'react-dom';
import ReservationCard from './ReservationCard';
import { shallow } from 'enzyme';

describe('ReservationCard', () => {

  it('should match the snapshot with all data passing correctly', () => {
    const wrapper = shallow(<ReservationCard
      date='12/5'
      id='123123'
      name='John Adams'
      number={4}
      time='4:00'
      key='123123'
      />);

    expect(wrapper).toMatchSnapshot();
  })

});
