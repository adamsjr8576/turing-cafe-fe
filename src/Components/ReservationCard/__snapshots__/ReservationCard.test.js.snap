// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReservationCard should match the snapshot with all data passing correctly 1`] = `
<article
  className="reservation-card"
>
  <h2>
    John Adams
  </h2>
  <p>
    12/5
  </p>
  <p>
    4:00
     PM
  </p>
  <p>
    Number Of Guest: 
    4
  </p>
  <button
    className="delete-button"
  >
    Cancel
  </button>
</article>
`;
